### DISCUSSION TITLE
A bit useless application ?

### DISCUSSION THREAD

**Discussion by circuitikzuser** [Tyrrrz/DiscordChatExporter#752] [Q&A] [answered]
Unless I understand something wrong, this application seems to be largely usesless if you want to back up your discord channel offline and e.g. delete the channel afterwards.

DiscordChatExporter unfortunately does not copy all the content over, and link most back to discord.
So all my images are still linked to discord after an archive has been created by DiscordChatExporter .
This is utterly useless.

So do I miss something or is this how DiscordChatExporter  works ?

**Comment by 96-LB** [comment, marked as answer]
Follow the steps in #733

**Comment by circuitikzuser** [comment]
Thanks I will try that.

**Comment by circuitikzuser** [comment]
Fantastic thanks. that solved it.
It looked like that particular window could not scroll down so I never saw the option at the bottom.
The download "attachments/images"  should be default, as I cannot see why anyone would want to archive their discord channel and still be dependent on Discord Site content.

**Reply by 96-LB** [reply]
The export is only a single file when media is not downloaded, making it easy to share. It also takes up far less space on the disk and far less bandwidth to download. Not everybody is looking to export channels that they plan to immediately delete, so it really just depends on what you want to do as there are pros and cons to both.

**Comment by Tyrrrz** [comment]
I see @96-LB already answered, but next time consider making your questions sound a bit less opinionated.

**Reply by circuitikzuser** [reply]
Get a life. Grow up, or desensitize yourself a bit: Did you read the title. ? I guess not.
